testsuite/makefiles/summarize.awk

```diff
@@ -157,19 +169,22 @@ END {
                     }else{
                         skips[skipidx++] = key;
                     }
+                }else if (r == "n"){
+                    ++ ignored;
                 }
             }
             printf("\n");
             printf("Summary:\n");
             printf("  %3d tests passed\n", passed);
             printf("  %3d tests skipped\n", skipped);
             printf("  %3d tests failed\n", failed);
+            printf("  %3d tests not started\n", ignored);
```

Comment:
What does "not started" mean, how does it compare to "skipped"?

Reply:
Thanks, this is very clear. You could incorporate the idea within the user report by using:

    printf("  %3d tests not started (parent test not passed)\n", ignored);

Reply:
That's a great idea - I went with "skipped or failed" rather than "not passed" as to me "not passed" very strongly implies failed, which of course most of the time it doesn't mean!